{"name": "test_today_wt", "status": "broken", "statusDetails": {"message": "AttributeError: 'Login' object has no attribute 'gp_login'", "trace": "driver_fixture = <appium.webdriver.webdriver.WebDriver (session=\"0e7b83d5-90bd-4243-ba0f-6351e0c81c1e\")>\n\n    @pytest.fixture(scope='session',autouse=False)\n    def gp_login_fixture(driver_fixture):\n        csv_file=BASE_DIR + '\\datas\\login.csv'\n        print(\"csv_file:%s\"%csv_file)\n        driver=driver_fixture\n        com=Common(driver)\n        data=com.get_csv_data(csv_file,2)\n        time.sleep(2)\n        trd=First(driver)\n        trd.to_login()\n        logn=Login(driver)\n>       logn.gp_login(data[0],data[1])\nE       AttributeError: 'Login' object has no attribute 'gp_login'\n\nconftest.py:54: AttributeError"}, "start": 1652795334159, "stop": 1652795334159, "uuid": "e7fd09e7-59df-4b13-97e2-4606ba5c79a5", "historyId": "a82216070e4d61dd73d80e9b1a457cf6", "testCaseId": "b2838e8a0659495958d6d7b6acdd13c5", "fullName": "testGp.test_new_Gp.TestGp#test_today_wt", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('back_fixture')"}, {"name": "parentSuite", "value": "testGp"}, {"name": "suite", "value": "test_new_Gp"}, {"name": "subSuite", "value": "TestGp"}, {"name": "host", "value": "Win102021AVUCSJ"}, {"name": "thread", "value": "10404-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testGp.test_new_Gp"}]}